src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        checkThreshold();
+        putValue(hash(key), key, value);
+        size++;
     }
 
     @Override
     public V getValue(K key) {
+        int index = hash(key);
+        Node<K, V> newNode = table[index];
+        while (newNode != null) {
+            if (Objects.equals(key, newNode.key)) {
+                return newNode.value;
+            }
+            newNode = newNode.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int hash(Object key) {
```

Comment:
```suggestion
    private int getIndex(Object key) {
```
method names should start with verb

Reply:
fixed